.gitignore

```diff
@@ -129,11 +129,12 @@ dmypy.json
 .pyre/
 
 # Assignment related files
-*.py
+# *.py
```

Comment:
Ideally we can just add a line to allow your report diff file and keep no other python files, I don't want anyone accidentally uploading their code :P

Reply:
Otherwise, thanks a bunch for adding to it, love the addition of the report diff!!